### PR TITLE
Rework E2E Tests (Remove STI, Test DNS)

### PR DESCRIPTION
**Pull request by DirectXMan12**
This PR replaces the STI parts of the E2E tests with a basic deployment of the hello-atomic example, and adds in a test for DNS.

### CODE REVIEW

**Comment by mheon** [comment]
Tests are passing on my system. Reviewing code at present.

File: examples/hello-atomic/all-in-one.tmpl.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+              port: 8080
+            timeoutSeconds: 1
+            initialDelaySeconds: 10
+        restartPolicy: Always
+        dnsPolicy: ClusterFirst
+        serviceAccount: ""
+parameters:
+- name: IMAGE
+  description: "The image to use"
+  value: atomicenterprise/hello-atomic:v0.5.2.2
```
Comment (aveshagarwal) [inline examples/hello-atomic/all-in-one.tmpl.yaml]: Is there any reason to go with hello-atomic:v0.5.2.2 when there is a later version available in the repo?

**Comment by mheon** [comment]
Code LGTM

File: examples/hello-atomic/all-in-one.tmpl.yaml (hunk 1 above)
Comment (DirectXMan12) [inline examples/hello-atomic/all-in-one.tmpl.yaml]: I was just following the style of the previous tests.  Also, it insulates against changes in the message breaking the tests, although that shouldn't be too much of a problem.

Comment (DirectXMan12) [inline examples/hello-atomic/all-in-one.tmpl.yaml]: fixed

**Comment by mheon** [comment]
@mbarnes I can't seem to get the E2E Docker tests running on my system - any chance you can run them to see if this is just me?

**Pull request closed without merging**